pkg/agent/util/net_windows.go

```diff
@@ -32,36 +34,74 @@ import (
 )
 
 const (
-	ContainerVNICPrefix = "vEthernet"
-	HNSNetworkType      = "Transparent"
-	LocalHNSNetwork     = "antrea-hnsnetwork"
-	OVSExtensionID      = "583CC151-73EC-4A6A-8B47-578297AD7623"
-	namedPipePrefix     = `\\.\pipe\`
+	ContainerVNICPrefix  = "vEthernet"
+	HNSNetworkType       = "Transparent"
+	LocalHNSNetwork      = "antrea-hnsnetwork"
+	OVSExtensionID       = "583CC151-73EC-4A6A-8B47-578297AD7623"
+	namedPipePrefix      = `\\.\pipe\`
+	commandMaxRetry      = 5
+	commandRetryInternal = time.Second
 )
 
 func GetNSPath(containerNetNS string) (string, error) {
 	return containerNetNS, nil
 }
 
+func GetHostInterfaceStatus(ifaceName string) (string, error) {
+	cmd := fmt.Sprintf(`Get-NetAdapter -InterfaceAlias "%s" | Select-Object -Property Status | Format-Table -HideTableHeaders`, ifaceName)
+	out, err := CallPSCommand(cmd)
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSpace(out), nil
+}
+
 // EnableHostInterface sets the specified interface status as UP.
 func EnableHostInterface(ifaceName string) error {
-	cmd := fmt.Sprintf("Enable-NetAdapter -InterfaceAlias %s", ifaceName)
-	return InvokePSCommand(cmd)
+	cmd := fmt.Sprintf(`Enable-NetAdapter -InterfaceAlias "%s"`, ifaceName)
+	// Enable-NetAdapter is not a block operation by our test. It returns immediately no matter if the interface
+	// has been enabled. So we need to check interface status to ensure the interface is up before return.
+	return wait.PollImmediate(commandRetryInternal, commandMaxRetry, func() (done bool, err error) {
+		if err := InvokePSCommand(cmd); err != nil {
+			klog.Errorf("Failed to run command %s: %v", cmd, err)
+			return false, nil
+		}
+		status, err := GetHostInterfaceStatus(ifaceName)
+		if err != nil {
+			klog.Errorf("Failed to run command %s: %v", cmd, err)
+			return false, nil
+		}
+		if !strings.EqualFold(status, "Up") {
+			klog.Errorf("Waiting for host interface %s to be up", ifaceName)
```

Comment:
Should this be INFO as it's a normal behavior?

Reply:
Done. Chaneg to INFO.